stacks/gcp/GcpStack.ts

```diff
@@ -122,6 +125,26 @@ export default class GcpStack extends TerraformStack {
       });
     }
 
+    // Create Cloud Tasks queue if it doesn't exist already
+    if (func.type === "queue" && !this.existingQueues.includes(func.name)) {
+      new CloudTasksQueue(this, func.name + "-queue", {
```

Comment:
Just thinking, would there be any benefit to make this a bit more flexible and allow users to override the queue name if they want to?

Reply:
Hey @tgerulaitis, a previous commit in this PR actually had the ability to enter in a name, but my latest commit removed it.

Messages passed into Cloud Tasks queues need the URL of the handler function passed in as part of the payload. By making both the name of the queue and the name of the function the same, it makes it easier to derive the destination URL and the fully qualified queue name.

(FYI for clarity, I'm concurrently working on some closed-source Space 48 company software that makes it easier to push messages to queues, and this PR is tied to being able to reference both the function URL and the queue name using one string parameter)

This is also the reason I changed the function name regex code in this PR, so that name generation follows all the allowed character conventions for these products.

My thinking is that if a developer really wants to add an additional queue with a custom queue name, then they can handwrite a CDKTF resource block and attach it to the infrastructure stack via the Cloud Seed JavaScript API.